### PR TITLE
Updated Readme, removed spurious lines I myself added on first commit.

### PR DESCRIPTION
**Pull request by misterybodon**
So it seems we cant merge right away, as I'm removing a few lines, but yet it seems no real conflict.

I've put up a local page and everything seems to run fine.

### CODE REVIEW

File: assets/js/contributorsPanel.js [hunk 1]
```diff
@@ -17,8 +17,8 @@ const contributors = [
 			displayName: "Mamun Abdullah",
 		},
 		{
-			githubName: "santimirandarp",
-			displayName: "Santiago Miranda",
```
Comment (DavidMatthewFraser) [inline assets/js/contributorsPanel.js]: I take it you own both santimirandarp and mysterbdon.

File: README.md [hunk 2]
```diff
@@ -1,34 +1,42 @@
...
+### Github Runs Git
+Something a bit confusing is that Github itself uses Git to keep track of the remote project. The more we understand our local copy, the more we will understand Github. 
 ## Steps to collaborate on this repo
-Basically you make a copy of this repository in your own GitHub account, add whatever changes you want, and then ask the person who created the repo to accept your changes by making a "pull request". That's how it works if the repo isn't yours. If you want to make your own changes and you don't want to change the original repo, you can work on your own fork.
 * Fork the repository (first you need to create a GitHub account)
 * Clone it from your fork to your local storage (`git clone https://github.com/yourUserHere/easy-to-git`)
 * Push to your repository on GitHub.
 * Log into GitHub and when you're ready, make a pull request. (Or do a pull request from your local machine?)   Do we even need to fork first??
 
+### A few notes
```
Comment (DavidMatthewFraser) [inline README.md:19]: This explanation is very appreciated! Thanks for taking the time 😄.

File: README.md [hunk 3]
```diff
@@ -1,34 +1,42 @@
...
+We end up updated.
+
 ## What changes do we expect in the files?
 Here we'll add some examples...
 - add your name to the contributers.md file
 - add your github name to the ``` contributors array ``` in the contributors.js 
 - add an html element, for example: ``` <h1>Open Source Rules</h1> ```
 - add some js, for example: ```logHello()```
 
+## Overwhelmed by commands?
```
Comment (DavidMatthewFraser) [inline README.md:36]: Looks like an amazing resource! I like that you really care about making this repo welcoming for new contributors 😊

File: README.md [hunk 4]
```diff
@@ -1,34 +1,42 @@
...
-
 # Hello and Welcome
 ## It should be a fun time doing the codez with all yall!
 ### You can check out tutorials such as <a href='https://github.com/firstcontributions/first-contributions/blob/master/README.md'>this one</a> to get famaliar with the git hubz
 
-
 # Easy to git
-The idea of this repo is to get you started with git and GitHub.
-
+The idea of this repo is to get you started with Git and GitHub.
```
Comment (DavidMatthewFraser) [inline README.md:6]: Good Catch

File: README.md [hunk 5]
```diff
@@ -1,34 +1,42 @@
...
 ## The very basics
-git and GitHub are different tools but work together. git is a program you run on your machine. And you use git to make and get changes to code stored on GitHub. GitHub is a remote host of files, that you work on with other people.
-
-### So can you repeat, how are those different?
-Because GitHub is on a different computer, when we create our remote repository, we set permissions for people to see the code, make copies or "fork" it, and "push" our changes to it.
-So the remote repository automatically becomes a collaboration tool. While git controls the project, we need a remote repository to collaborate, and that is GitHub.
-
+Git and GitHub are different tools but work together. Git is a program you run on *your machine*, and it's useful to interact with the remote code-repository. This remote host of files is GitHub.
```
Comment (DavidMatthewFraser) [inline README.md:8]: You really get it. Thanks for putting so much thought into cutting down on bulk and making clear explanations.

File: assets/img/quote-2.svg [hunk 6]
```diff
@@ -1,47 +1 @@
...
-<g>
-</g>
-<g>
-</g>
-<g>
-</g>
-<g>
-</g>
-</svg>
+<svg xmlns="http://www.w3.org/2000/svg" width="508.044" height="508.044"><path d="M507.93 155.673c0-.055.006-.11.006-.165 0-66.793-54.145-120.938-120.938-120.938S266.061 88.714 266.061 155.508c0 66.794 54.15 120.938 120.938 120.938 13.727 0 26.867-2.393 39.162-6.609-27.209 156.09-148.93 256.752-36.096 173.905C515.182 351.874 508.07 159.198 507.93 155.673zM120.938 276.445c13.727 0 26.867-2.393 39.168-6.609-27.216 156.09-148.937 256.752-36.102 173.905 125.117-91.867 118.006-284.543 117.865-288.068 0-.055.006-.11.006-.165 0-66.793-54.144-120.938-120.937-120.938C54.144 34.57 0 88.714 0 155.508c0 66.794 54.15 120.937 120.938 120.937z"/></svg>
```
Comment (DavidMatthewFraser) [inline assets/img/quote-2.svg:1]: Thanks for cleaning this up!

File: index.html [hunk 7]
```diff
@@ -34,10 +34,6 @@ <h3>Easy to HTML</h3>
       <main class="main">
       
         <div class="my-container">
-        <h2 class="greeting">Hello from Nowhereland :-)</h2>
```
Comment (DavidMatthewFraser) [inline index.html:51]: The h2 an p had a good run 😄

**Comment by santimirandarp** [comment]
you are welcome ;-)

will starting getting into the code now

**Pull request merged**